package-lock.json

```diff
@@ -7358,9 +7366,110 @@
 				"react-dates": "^17.1.1",
 				"react-spring": "^8.0.20",
 				"rememo": "^3.0.0",
-				"styled-components": "^4.4.0",
 				"tinycolor2": "^1.4.1",
 				"uuid": "^3.3.2"
+			},
+			"dependencies": {
```

Comment:
The lock file needs some tweaks, to ensure that emotion gets hoisted. The easiest way is to install it in the root of the project and then remove it from `package.json` file 😅

Reply:
@gziolo oh! Okay. I didn't know that. I'll give it a shot!